springapp/src/test/java/com/javaacademy/robot/controllers/BookRestAPIControllerTest.java

```diff
@@ -49,45 +51,75 @@ public void shouldReturnOneBook() {
 
     @Test
     public void shouldReturnEmptyBook() {
-        BookSearch bookSearchMock = mock(BookSearch.class);
-        BookRestAPIController bookRestAPIController = new BookRestAPIController(null, bookSearchMock);
+        BookService bookServiceMock = mock(BookService.class);
+        BookRestAPIController bookRestAPIController = new BookRestAPIController(bookServiceMock);
         String givenQuery = "Franklin";
-        when(bookSearchMock.search(givenQuery)).thenReturn(Collections.emptyList());
+        when(bookServiceMock.searchOneKeyword(givenQuery.toLowerCase())).thenReturn(getBookDtos());
         List<BookDto> givenSearchResults = bookRestAPIController.searchResults(givenQuery);
 
-        assertEquals(Collections.emptyList(), givenSearchResults);
+        assertEquals(2, givenSearchResults.size());
     }
```

Comment:
In all subsequent tests you have formatting convention to leave one blank line between 'given', 'when' and 'then' section. In all previous tests you have it only before assertion. You need to be consistent :)